lib/le.rb

```diff
@@ -44,22 +42,16 @@ def self.new(token, region, options={})
     logger
   end
 
-  def self.checkParams(token, region, opt_datahub_enabled, opt_udp_port)
-    # Check if the key is valid UUID format
+  def self.check_params(token, region, opt_datahub_enabled, opt_udp_port)
+    # test Token only when DataHub and UDP are not enabled
+    return unless !opt_datahub_enabled && !opt_udp_port
 
-    if (!opt_datahub_enabled && !opt_udp_port)  # test Token only when DataHub and UDP are not enabled
-      if (token =~ /\A(urn:uuid:)?[\da-f]{8}-([\da-f]{4}-){3}[\da-f]{12}\z/i) == nil
-         puts "\nLE: It appears the LOGENTRIES_TOKEN you entered is invalid!\n"
-      else
-        (token="")
-      end
-
-      # This checks if region is valid. So far we have Europe and US
-      if !region
-        puts "\nLE: You need to specify region. Options: eu, us"
-      end
+    # Check if the key is valid UUID format
+    if (token =~ /\A(urn:uuid:)?[\da-f]{8}-([\da-f]{4}-){3}[\da-f]{12}\z/i).nil?
+      puts "\nLE: It appears the LOGENTRIES_TOKEN you entered is invalid!\n"
+    end
 
-   end
+    # This checks if region is valid. So far we have Europe and US
+    puts "\nLE: You need to specify region. Options: eu, us" unless region
```

Comment:
Let's remove the references to EU and US as we now have several more. Or even:
```suggestion
    puts "\nLE: You need to specify region, such as 'eu'" unless region
```

And the comment above to be changed

Reply:
implemented